article.html

```diff
@@ -0,0 +1,437 @@
+          smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.</p>
+        <p class="paragraph1">Inflation has been the workhorse of cosmology for 35 years, though many, including Dr.
+          Guth, wondered whether it could ever be proved.</p>
+        <p class="paragraph1">If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the
+          recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast
+          realms of time and space and energy to science and speculation.</p>
+        <p class="paragraph1">Confirming inflation would mean that the universe we see, extending 14 billion light-years
+          in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose
+          extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless
+          number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+        <p class="paragraph1"><strong class="paragraphtitle">'As Big as it Gets'</strong></p>
+        <p class="paragraph1">In our own universe, it would serve as a window into the forces operating at energies
+          forever beyond the reach of particle accelerators on Earth and yield new insights into gravity itself. Dr.
+          Kovac’s ripples would be the first direct observation of gravitational waves, which, according to Einstein’s
+          theory of general relativity, should ruffle space-time.</p>
+        <p class="paragraph1">Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part
+          of the team, said, “This is huge, as big as it gets.”</p>
+        <p class="paragraph1">He continued, “This is a signal from the very earliest universe, sending a telegram
+          encoded in gravitational waves.”</p>
+        <p class="paragraph1">The ripples manifested themselves as faint spiral patterns in a bath of microwave
+          radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as
+          hot as the surface of the sun.</p>
+        <p class="paragraph1">Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background
+          Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center
+          for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+      </div>
+      <div class="image_center">
+        <h2>The Theory of Inflation</h2>
+        <span class="caption">Astronomers have found evidence to support the theory of inflation, which explains how the
+          universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</span>
+            
```

Comment:
## HTML validator
- Check this  line it's throwing some errors
![image](https://user-images.githubusercontent.com/24902525/104540287-2584c380-55ed-11eb-9137-8ad05fbbbb11.png)